Update links to supported middleware topics

lancedouglas1 opened the pull request:
The the pre-processing link was incorrect (pointed at static), 
but there also doesn't seem to be an accurate 'dynamic
request handling' example.  I'll add one once I figure it out
in loopback if someone doesn't already.

Commit:
Update links to supported middleware topics

The the pre-processing link was incorrect (pointed at static), but there also doesn't seem to be an accurate 'dynamic request handling' example.  I'll add one once I figure it out in loopback if someone doesn't already.

slnode commented:
Can one of the admins verify this patch?

dhmlau commented:
@lancedouglas1 , thanks for your PR.  It seems like there's linting error in your commit message.  Could you please fix your commit message?  Thanks. 

```
**  Linting commit logs
**
**  1 problems found:
**    fb7a33b - Update links to supported middleware topics: line 3 longer than 72 characters.
```

Commit:
Update links to supported middleware topics (#1)

The the pre-processing link was incorrect (pointed at static),
but there also doesn't seem to be an accurate 'dynamic
request handling' example. I'll add one once I figure it out
in loopback if someone doesn't already.

slnode commented:
Can one of the admins verify this patch?
